README.md

```diff
@@ -36,6 +36,7 @@ Options are used to change the behavior.
 | -addr string        | address to listen on ip:port (default ":9200")                                                |
 | -clusteruuid string | Cluster UUID of Elasticsearch we are mocking, needed if beat is being monitored by metricbeat |
 | -metrics duration   | Interval to print metrics to stdout, 0 is no metrics                                          |
+| -delay duration     | Golang Duration to wait before processing API request, 0 is no delay                          |
```

Comment:
It might be a bit of a nit-pick, but it's Go, not Golang, also referencing `time.Duration` directly looks more clear to me. What do you think?

```suggestion
| -delay duration     | Go `time.Duration` to wait before processing API request, 0 is no delay                          |
```

The same goes to the CLI help message.

Reply:
I likeit. I added it to both `-delay` and `-metrics`.